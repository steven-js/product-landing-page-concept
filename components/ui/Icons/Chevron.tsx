const Chevron = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="10"
      height="16"
      fill="none"
      viewBox="0 0 10 16"
    >
      <path
        fill="var(--neutral-neutral-1000)"
        d="m9.275 8.225-7.5 7.5-1.05-1.05 6.45-6.45-6.45-6.45 1.05-1.05 7.5 7.5Z"
      />
    </svg>
  )
}

export default Chevron
