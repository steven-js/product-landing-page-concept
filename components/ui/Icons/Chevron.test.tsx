import { render } from '@testing-library/react'
import Chevron from './Chevron'

describe('Chevron icon test', () => {
  it('should render the Chevron icon unchanged', () => {
    const { container } = render(<Chevron />)
    expect(container).toMatchSnapshot()
  })
})
