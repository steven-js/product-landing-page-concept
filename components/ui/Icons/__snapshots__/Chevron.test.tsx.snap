// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chevron icon test should render the Chevron icon unchanged 1`] = `
<div>
  <svg
    fill="none"
    height="16"
    viewBox="0 0 10 16"
    width="10"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="m9.275 8.225-7.5 7.5-1.05-1.05 6.45-6.45-6.45-6.45 1.05-1.05 7.5 7.5Z"
      fill="var(--neutral-neutral-1000)"
    />
  </svg>
</div>
`;
